docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,46 +24,90 @@ The following table shows how risk levels are applied to a host, based on the no
-+
+image::images/enable-hrs.png[Enable Host Risk Score button]
+
+To enable host risk score from a host's details page: 
+
+. Go to *Explore* -> *Hosts*. 
+. Select the *All hosts* tab, then click a host name. 
+. On the details page, scroll down to the data tables, then select the *Host risk* tab. 
+. Click *Enable* to install the module. 
+
 [role="screenshot"]
-image::images/host-risk-score-dev-tools-console.png[]
+image::images/enable-hrs-details-pg.gif[Shows how to enable host risk score from the host's details page]
 
-[[import-host-risk-score-dashboard]]
+To enable host risk score from Dev Tools in {kib}, open a browser window and enter the following URL:
+
+[source,console]
+----------------------------------
+{KibanaURL}/s/{spaceID}/app/dev_tools#/console?load_from={KibanaURL}/s/{spaceID}/internal/risk_score/prebuilt_content/dev_tool/enable_user_risk_score
+----------------------------------
+
+
+NOTE: After enabling host risk score, you may get a message that says, "No host risk score data available to display." You may need to wait about an hour for data to be generated, or you can click *Restart* to force host risk score calculations to run immediately. 
+
+[[upgrade-host-risk-score]]
 [discrete]
-=== Import the host risk score dashboard
+=== Upgrade host risk score 
 
-After you have set up the framework for the host risk score feature, import pre-built dashboards to analyze risky hosts in your environment:
+If you previously enabled host risk score in {stack} versions 8.4 and earlier and are upgrading to version 8.5 or later, there'll be an option to "upgrade" host risk score instead of enabling it. 
 
-. Navigate to the *Current host risk scores* card (*Dashboards -> Overview*).
-. Click *Import dashboard*.
-+
-[role="screenshot"]
-image::images/host-risk-score-import-dashboard.png[]
+Before upgrading, be mindful of the following: 
+
+* Previous host risk scores are deleted since the older data is not preserved, and new scores are created. However, if you want to retain the old host risk scores, you'll need to reindex the data _before_ upgrading the feature. Your data will then be stored in the `ml_{host|user}_risk_score_{spaceId}` and `ml_${host|user}_risk_score_latest_{spaceID}` indices. To learn how to do this, refer to {ref}/docs-reindex.html[Reindex API]. 
+
+* You must edit your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and remove the `xpack.securitySolution.enableExperimental:['riskyHostsEnabled']` feature flag. 
 
-[[view-host-risk-score]]
+After this is done, you can proceed with upgrading the host risk score feature from any of the following places in the {security-app}:
+
+* The Entity Analytics dashboard 
+* The *Host risk* tab on the Hosts page
+* The *Host risk* tab on a host's details page
+
+[[import-host-risk-score-dashboard]]
 [discrete]
-=== View host risk score data
+=== Import the host risk score dashboard (Optional)
```

Comment:
The dashboards will be automatically imported during upgrade / installation process, so there's no need for users import dashboards manually. I'd recommend to re-name `Import the host risk score dashboard` to something like Explore the host risk score dashboard